### DISCUSSION TITLE
Inaccuracies in star azimuth/altitude within Plaeides star cluster

### DISCUSSION THREAD

**Discussion by GreatPilot8** [Stellarium/stellarium#3582] [Q&A]
Good morning,

I'm programming a simulation for a school project which displays various stars and their positions in the night sky on December 21st 2022 0000EST (my birthday). When I add the azimuth and altitude values for stars in the Pleiades star cluster from the Stellarium Windows app, I get a strange shape in return:
![image](https://github.com/Stellarium/stellarium/assets/42357259/91d465c7-e449-440a-af7b-e5a2ebc22903)
Instead of:
![image](https://github.com/Stellarium/stellarium/assets/42357259/3cef6f16-c70c-42ae-b4dc-49860f15dcec)
The constellations I modelled from the Stellarium data appear correct but they are also much larger than the star cluster. Is this an inaccuracy on my part or Stellarium?

Thank you in advance for your help!

**Comment by gzotti** [comment]
I don't know how you created your first, smaller image. With a few million users, I think an error in the Pleiades star cluster in Stellarium would have been reported by around 2004 or so, and about weekly ever since. Note that e.g. there are 60 arcminutes in the degree.

**Reply by GreatPilot8** [reply]
Thank you for your reply. The error was that I am stupid. I've got it all sorted now!
Cheers!
